src/lando/ui/legacy/forms.py

```diff
@@ -147,24 +148,10 @@ def clean_revision_ids(self) -> list[int]:
         """Parse and validate the comma-separated revision IDs."""
         revision_ids_str = self.cleaned_data["revision_ids"]
 
-        if not revision_ids_str:
-            raise forms.ValidationError("At least one revision ID is required.")
-
         try:
-            revision_ids = [
-                int(rev_id.strip())
-                for rev_id in revision_ids_str.split(",")
-                if rev_id.strip()
-            ]
-        except ValueError:
-            raise forms.ValidationError(
-                "Invalid revision IDs. Must be comma-separated integers."
-            )
-
-        if not revision_ids:
-            raise forms.ValidationError("At least one revision ID is required.")
-
-        return revision_ids
+            return parse_revision_ids(revision_ids_str)
+        except ValueError as e:
+            raise forms.ValidationError(str(e))
```

Comment:
I'd suggest 

```suggestion
            raise forms.ValidationError(str(e)) from e
```

so we can get exception chains if they aren't caught.